applications/admin.py

```diff
@@ -166,43 +165,44 @@ def get_queryset(self, request):
             .get_queryset(request)
             .prefetch_related(
                 "app_step_submissions__bootcamp_application__user",
-                "app_step_submissions__bootcamp_application__bootcamp_run",
+                "app_step_submissions__bootcamp_application__bootcamp_run__bootcamp",
             )
         )
 
     def get_user_email(self, obj):
         """Returns the user email"""
-        app_step_submission = obj.app_step_submissions.first()
+        app_step_submissions = obj.app_step_submissions.all()
         return (
             None
-            if app_step_submission is None
-            else app_step_submission.bootcamp_application.user.email
+            if len(app_step_submissions) == 0
+            else app_step_submissions[0].bootcamp_application.user.email
         )
 
     get_user_email.short_description = "Application User"
     get_user_email.admin_order_field = (
         "app_step_submission__bootcamp_application__user__email"
     )
 
-    def get_run_title(self, obj):
-        """Returns the bootcamp run title"""
-        app_step_submission = obj.app_step_submissions.first()
+    def get_run_display_title(self, obj):
+        """Returns the bootcamp run display title"""
+        app_step_submissions = obj.app_step_submissions.all()
         return (
             None
-            if app_step_submission is None
-            else app_step_submission.bootcamp_application.bootcamp_run.title
+            if len(app_step_submissions) == 0
+            else app_step_submissions[0].bootcamp_application.bootcamp_run.display_title
         )
 
-    get_run_title.short_description = "Application Bootcamp Run"
-    get_run_title.admin_order_field = (
+    get_run_display_title.short_description = "App Bootcamp Run"
+    get_run_display_title.admin_order_field = (
         "app_step_submission__bootcamp_application__bootcamp_run__title"
     )
 
     def app_step_submission_link(self, obj):
         """Returns a link to the related bootcamp application"""
-        app_step_submission = obj.app_step_submissions.first()
-        if app_step_submission is None:
+        app_step_submissions = obj.app_step_submissions.all()
```

Comment:
why this change?

Reply:
Takes advantage of the prefetched queryset. If you use `first()`, that runs an extra query.